packages/vm/test/api/EIPs/eip-7623.spec.ts

```diff
@@ -86,7 +86,7 @@ describe('EIP 7623 calldata cost increase tests', () => {
       })
       assert.fail('runTx should throw')
     } catch {
-      assert.ok('Successfully failed')
+      assert.isTrue(true, 'Successfully failed')
```

Comment:
Conversion candidate for expect or assert.doesNotThrow?

Reply:
im not sure if this works for async functions.  it caused a vitest error for me and i can't find an example of it with an async function anywhere

Reply:
Have pushed a fix and verified it works.  If you change the regex message, the test will fail so it's correctly checking the error message.